src/cli.ts

```diff
@@ -31,7 +31,7 @@ import publish from './install/publish';
 import { getBin } from './install/bin';
 import { spawn } from 'child_process';
 
-const installEqualRegEx = /^(@?([-_\.a-z\d]+\/)?[\-\_\.a-z\d]+)=/i;
+const installEqualRegEx = /^(@?([-_\.a-z\d]+\/)?[\-\_\.a-z\d]+(@[-_\.a-z\d]+)?)=/i;
```

Comment:
Perhaps we should just simplify this entirely to just `/^[-_\.a-z\d@]+=/i`?

Reply:
I tried to keep it as strict as it was before.  If `/^[-_\.a-z\d@]+=/i` looks good to you I can make that change and update the PR.